cmd/podman/root.go

```diff
@@ -89,7 +94,7 @@ func init() {
 		syslogHook,
 		earlyInitHook,
 		configHook,
-		noOutHook,
+		stdOutHook,
```

Comment:
I'll definitely do this, but to clarify the only reason stdout is being captured is because a number of podman's commands have an option to format its output as json, or they output just an id. Stderr isn't well-formed (and making it so, seems like it'd be more work on your part), so it's not really parseable.

Originally, I wasn't planning on submitting a PR and was going to come up with a workaround, but then I saw that someone contributed --noout which is remarkably similar logic. So, essentially this supersedes --noout since it provides the same functionality (and fixes a bug with --noout), but I'm leaving that up to you guys since --noout was already contributed.

Reply:
Updated the order, but included a comment cautioning the reader about how the Stdout hook may affect the output of any of the hooks that follow it. This way if someone decides they want to reuse the hook to capture the plaintext Stderr output of podman instead of checking the exit code, it'll (hopefully) be apparent that they'll be affecting the error output of critical global state like logging, rlimit failures, or syslog constraints..